Configuration/Sets/Weather2/config.yaml

```diff
@@ -0,0 +1,2 @@
+name: jweiland/weather2
+label: Weather2 Extension Settings
```

Comment:
"Extension Setting" is a defined TYPO3 wording for BE module "Settings" -> "Extension settings". This Site Set has nothing to do with extension settings. Can you please change that label?